src/ray/raylet/node_manager.cc

```diff
@@ -286,7 +288,8 @@ void NodeManager::Heartbeat() {
   auto heartbeat_data = std::make_shared<HeartbeatTableData>();
   SchedulingResources &local_resources = cluster_resource_map_[self_node_id_];
   heartbeat_data->set_client_id(self_node_id_.Binary());
-  // TODO(atumanov): modify the heartbeat table protocol to use the ResourceSet directly.
+  // TODO(atumanov): modify the heartbeat table protocol to use the ResourceSet
```

Comment:
hmmm. maybe revert this? is this due to clangformat version incompatibility or some editor configuration?

Reply:
if you didn't change anything, I would look into the `.editorconfig` file that's recently added